exporters/otlp/test/otlp_http_exporter_test.cc

```diff
@@ -158,34 +157,35 @@ class OtlpHttpExporterTestPeer : public ::testing::Test
     auto mock_session =
         std::static_pointer_cast<http_client::nosend::Session>(no_send_client->session_);
     EXPECT_CALL(*mock_session, SendRequest)
-        .WillOnce([&mock_session, report_trace_id](
-                      std::shared_ptr<opentelemetry::ext::http::client::EventHandler> callback) {
-          auto check_json =
-              nlohmann::json::parse(mock_session->GetRequest()->body_, nullptr, false);
-          auto resource_span     = *check_json["resourceSpans"].begin();
-          auto scope_span        = *resource_span["scopeSpans"].begin();
-          auto span              = *scope_span["spans"].begin();
-          auto received_trace_id = span["traceId"].get<std::string>();
-          EXPECT_EQ(received_trace_id, report_trace_id);
-
-          auto custom_header = mock_session->GetRequest()->headers_.find("Custom-Header-Key");
-          ASSERT_TRUE(custom_header != mock_session->GetRequest()->headers_.end());
-          if (custom_header != mock_session->GetRequest()->headers_.end())
-          {
-            EXPECT_EQ("Custom-Header-Value", custom_header->second);
-          }
-
-          auto user_agent_header = mock_session->GetRequest()->headers_.find("User-Agent");
-          ASSERT_TRUE(user_agent_header != mock_session->GetRequest()->headers_.end());
-          if (user_agent_header != mock_session->GetRequest()->headers_.end())
-          {
-            EXPECT_EQ(GetOtlpDefaultUserAgent(), user_agent_header->second);
-          }
-
-          // let the otlp_http_client to continue
-          http_client::nosend::Response response;
-          response.Finish(*callback.get());
-        });
+        .WillOnce(
+            [&mock_session, report_trace_id](
```

Comment:
Why is the code formatted differently ?

All the formatting is already checked in CI, with clang-format.

Not sure if/why clang-tidy changed it, or if this is the result of running a possible different version of clang-format, but it will be better if clang-tidy avoids touching the code format at all.

We can't have multiple tools compete on this, generating noisy diffs.